exporter/instanaexporter/README.md

```diff
@@ -0,0 +1,73 @@
+# Instana Exporter
+
+| Status                   |                  |
+| ------------------------ |------------------|
+| Stability                | [beta]           |
```

Comment:
Please, review the possible values and select accordingly. For this PR, it should probably be "in development", switching to "alpha" once the component is ready. "beta" implies that this component has had at least "some" exposure to production workloads.

Reply:
Agree, updated.